packages/funding-rate-proposer/src/proposer.js

```diff
@@ -98,7 +98,7 @@ class FundingRateProposer {
     ]);
   }
 
-  async updateFundingRates() {
+  async updateFundingRates(useRealTime = false) {
```

Comment:
Might be worth a note here about what environments useRealTime is set to true for.